### ISSUE TITLE
Ansible Service Broker is broken with default image in 3.10

### ISSUE THREAD

**Issue by eriknelson** [openshift/openshift-ansible#9960]
#### Description

Provide a brief description of your issue here. For example:

By default, 3.10 will deploy the latest asb image from upstream: https://github.com/openshift/openshift-ansible/blob/release-3.10/roles/ansible_service_broker/defaults/main.yml#L26

This image is targeted for 3.11 and will not work. In 3.10, the broker mounts its router behind a path of `/ansible-service-broker/`, while in 3.11, the broker expects `/osb/`. The default value in openshift-ansible should deploy a broker specifically built for 3.10 and not follow a floating latest, which is built from the broker master. The latest 3.10 image is: `docker.io/ansibleplaybookbundle/origin-ansible-service-broker:ansible-service-broker-1.2.17-1`.

##### Steps To Reproduce
Deploy a cluster with ansible-service-broker enabled, and accept the default origin broker image.

##### Expected Results
The service catalog should be able to contact the broker successfully.

##### Observed Results
The service catalog 404s when requesting the broker's catalog. The following command will display the 404 error under status:

`oc describe clusterservicebroker $ASB_BROKER_NAME`

**Comment by cmoulliard** [comment]
What is the workaround when we install the `openshift-service-catalog` playbook ? To pass as parameter the following key/value `-e ansible_service_broker_registry_tag=release-1.2` to be sure to install the version which is working with origin 3.10 ? @eriknelson

**Comment by eriknelson** [comment]
@cmoulliard The key role is the `ansible_service_broker` role. The `openshift_service_catalog` role does not install any brokers, but the catalog itself afaik. This only affects upstream origin installs, as the downstream install will pull the correctly tagged broker image. If you are running an upstream origin install, overriding `ansible_service_broker_image: "docker.io/ansibleplaybookbundle/origin-ansible-service-broker:ansible-service-broker-1.2.17-1"` should deploy our latest upstream ansible broker image.

**Comment by cmoulliard** [comment]
Many thanks for the explanation and clarification. We will make a test using this parameter `ansible_service_broker_image`

**Comment by jwmatthews** [comment]
cc @jmontleon

**Comment by maverick85** [comment]
Hi there,

I would like to say I came across this issue in a brand new clean setup.
After deploying cluster with deploy_cluster.yml, I noticed openshift-ansible-service-broker had failed to deploy.
So I edited and changed the image from:
`docker.io/ansibleplaybookbundle/origin-ansible-service-broker:v3.11.0`
to
`docker.io/ansibleplaybookbundle/origin-ansible-service-broker:ansible-service-broker-1.2.17-1`

And it got fixed. But this issue is still live.

```
# oc version
oc v3.11.0+62803d0-1
kubernetes v1.11.0+d4cacc0
features: Basic-Auth GSSAPI Kerberos SPNEGO

openshift v3.11.0+e5dbec2-186
kubernetes v1.11.0+d4cacc0
```

**Comment by maverick85** [comment]
Hello again guys.

I reinstalled from scratch I am using a 3 server setup, and I added
`ansible_service_broker_image=docker.io/ansibleplaybookbundle/origin-ansible-service-broker:ansible-service-broker-1.2.17-1`
to the inventory file, which went perfectly thru.

I was wondering how to submit a change request for the documentation, as the online documentation doesn't provide information about this, so how to request to add to documentation? Thank you

https://docs.openshift.com/container-platform/3.11/install/configuring_inventory_file.html#configuring-openshift-ansible-broker
